plugins/chat/db/migrate/20221122070108_save_chat_allowed_groups_site_setting.rb

```diff
@@ -6,7 +6,7 @@ def up
     return if chat_enabled.blank?
 
     chat_allowed_groups = DB.query_single("SELECT value FROM site_settings WHERE name = 'chat_allowed_groups'")
-    return if chat_allowed_groups.blank?
+    return if chat_allowed_groups.present?
```

Comment:
We _don't_ want to proceed if the `chat_allowed_groups` setting was already saved, not if it was blank.